fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/AvatarTokens.kt

```diff
@@ -694,6 +695,12 @@ open class AvatarTokens(private val activityRingToken: ActivityRingsToken = Acti
         }
     }
 
+    @Composable
+    open fun cutoutColorFilter(avatarInfo: AvatarInfo): ColorFilter? {
+        return null
```

Comment:
Would it be helpful for developers if we add a default colorFilter for the token?

Reply:
by default it will show the image vector as it is
only if you want to spcify any color filter then you should add